### PR TITLE
[S] RR: create big Rating number and stars feature (Ticket#115)

### PR DESCRIPTION
**Pull request by squeakypickles33**

### CODE REVIEW

File: client/src/global.css [hunk 1]
```diff
@@ -561,3 +561,7 @@ TODO: override default styles
   color: blue;
   text-decoration: underline;
 }
+
+.rr-average-rating {
+  font-size: 50px;
```
Comment (burhan-syed) [inline client/src/global.css:566]: Can you convert this to rem. 50px is about 3.8rem with our current base font size.

File: client/src/components/RatingsAndReviews/ProductBreakdown.jsx [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+const ProductBreakdown = ({reviewMeta}) => {
+  const ratings = reviewMeta.ratings;
+  const ones = parseInt(ratings["1"]);
+  const twos = parseInt(ratings["2"]);
+  const threes = parseInt(ratings["3"]);
+  const fours = parseInt(ratings["4"]);
+  const fives = parseInt(ratings["5"]);
+  const totalScore = ones + twos*2 + threes*3 + fours*4 + fives*5;
+  const totalRatings = ones + twos + threes + fours + fives;
+  const average = Math.round(totalScore/totalRatings*10)/10;
```
Comment (burhan-syed) [inline client/src/components/RatingsAndReviews/ProductBreakdown.jsx:13]: .toFixed might've also worked here

**Review by burhan-syed — APPROVED** [review]

File: client/src/global.css (hunk 1 above)
Comment (squeakypickles33) [reply client/src/global.css:566]: yes ill do that now

File: client/src/components/RatingsAndReviews/ProductBreakdown.jsx (hunk 2 above)
Comment (squeakypickles33) [reply client/src/components/RatingsAndReviews/ProductBreakdown.jsx:13]: I actually was thinking of doing that but would have been my first time using it, ill try it

**Pull request merged**